src/databricks/labs/remorph/snow/sql_transpiler.py

```diff
@@ -15,13 +18,30 @@ def __init__(self, read_dialect: Dialect):
     def transpile(
         self, write_dialect: Dialect, sql: str, file_name: str, error_list: list[ParserError]
     ) -> TranspilationResult:
+        transpiled_sql_statements = []
         try:
-            transpiled_sql = transpile(sql, read=self.read_dialect, write=write_dialect, pretty=True, error_level=None)
+            parsed_expressions = parse(sql, read=self.read_dialect, error_level=ErrorLevel.WARN)
```

Comment:
Why is the change in error level requires changing and is all the expression iteration needed?